language/testing-infra/e2e-tests/src/executor.rs

```diff
@@ -315,6 +315,7 @@ impl FakeExecutor {
             let vm = MoveVM::new();
             let remote_view = RemoteStorage::new(&self.data_store);
             let mut session = vm.new_session(&remote_view);
+            let logger = NoLogLogger;
```

Comment:
I am not sure a no logger is right here, and have to play with it a bit to see what is the best solution.
We may change later. Opinions welcome.
Overall for tests it's not clear any logging is meaningful as if there are issues it's for the test to assert them...